docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -152,7 +152,7 @@ BigDL-LLM for GPU supports on Linux has been verified on:
 
 ```
 
-### Install BigDL-LLM From PyPI
+### Install BigDL-LLM And Dependencies From PyPI
```

Comment:
No need to change this? I think Install BigDL-LLM From Pypi is fine

Reply:
Reverted back to "Install BigDL-LLM From PyPI"

Reply:
have you reverted?

Reply:
It was reverted when I rebased my branch on top of main and force pushed, around the time of initial review. Somehow the revert is not displayed in the file diff.
Current file - https://github.com/chtanch/BigDL/blob/enhance-gpu-doc/docs/readthedocs/source/doc/LLM/Overview/install_gpu.md